smarts/core/sumo_road_network.py

```diff
@@ -86,10 +86,13 @@ def write_glb(self, output_path):
 
 
 class SumoRoadNetwork:
-    def __init__(self, graph, net_file):
+    def __init__(self, graph, net_file, default_lane_width=None):
         self._log = logging.getLogger(self.__class__.__name__)
         self._graph = graph
         self._net_file = net_file
+        self._default_lane_width = (
+            default_lane_width if default_lane_width is not None else 3.2
```

Comment:
should we add some explanation for 3.2 here?

Reply:
Yes, I will add a comment.

Reply:
ok, done.